python/tvm/micro/session.py

```diff
@@ -259,6 +259,8 @@ def compile_and_create_micro_session(
     mod_src_bytes: bytes,
     template_project_dir: str,
     project_options: dict = None,
+    build_dir: str = None,
+    debug: bool = False,
```

Comment:
could we use a more descriptive parameter name? e.g. `reuse_project` or `generate_project: bool = True`? alternatively, could derive this from `build_dir` (e.g. if it already exists, don't regenerate); but that could be troublesome if we want to assert a workflow actually generates a project too.

Reply:
how about `skip_project_generation: bool = False`? yes, we can drive it from `build_dir` but I think that might cause some confusion and it's better to explicitly express it when we would like to skip project generation.

Reply:
I like `reuse_project` or `use_existing`. I also agree that deriving this property from `build_dir` would cause confusion.

Reply:
I used `use_existing`. Thanks @guberti for the suggestion.